<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CertsEditControl</class>
 <widget class="QWidget" name="CertsEditControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="importButton">
     <property name="text">
      <string>インポート</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="showButton">
     <property name="text">
      <string>表示</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="deleteButton">
     <property name="text">
      <string>削除</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="filenameLabel">
     <property name="text">
      <string>Filename</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
